app/views/saml_idp/shared/saml_post_binding.html.erb

```diff
@@ -18,16 +20,17 @@
             <%= t('.no_js') %>
           </p>
 
-          <%= form_tag action_url, id: 'saml-post-binding' do %>
+          <%= simple_form_for('', url: action_url) do |f| %>
             <%= hidden_field_tag('csp_uris', csp_uris) if Rails.env.test? %>
```

Comment:
do we still need this hidden field for tests?

Reply:
> do we still need this hidden field for tests?

Maybe! I can check

Reply:
I think it can be removed. The last reference was removed in #3479.

Reply:
Removed unnecessary field in c928e52.